_config.php

```diff
@@ -1,5 +1,9 @@
 <?php
 
 use SilverStripe\Dev\Deprecation;
+use SilverStripe\ORM\DB;
 
 Deprecation::notification_version('4.0.0', 'tractorcow/silverstripe-fluent');
+
+// Credit to https://github.com/sunnysideup thank you for the mysql fix
+DB::query("SET SESSION sql_mode='REAL_AS_FLOAT,PIPES_AS_CONCAT,ANSI_QUOTES,IGNORE_SPACE';");
```

Comment:
Same problem as https://github.com/tractorcow-farm/silverstripe-fluent/pull/617

Need to conditional and apply only to mysql, not postgres / sqlite etc...